dev/com.ibm.ws.org.apache.cxf.cxf.core.3.2/src/org/apache/cxf/service/model/MessagePartInfo.java

```diff
@@ -0,0 +1,180 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.cxf.service.model;
+
+import javax.xml.namespace.QName;
+
+import org.apache.ws.commons.schema.XmlSchemaAnnotated;
+import org.apache.ws.commons.schema.XmlSchemaElement;
+import java.util.logging.Logger;
+import org.apache.cxf.common.logging.LogUtils;
+import java.util.logging.Level;
+
+public final class MessagePartInfo extends AbstractPropertiesHolder implements NamedItem {
```

Comment:
I think we shouldn't let trace injection be added to this class. It's one of the core pieces of CXF, and so trace injection will be logging this class like crazy. Can you add this class to the list of excludes in the `bnd.bnd`

Reply:
I agree, disabled trace injection for this class.